src/test/resources/sql/clean-up.sql

```diff
@@ -1,4 +1,5 @@
 SET FOREIGN_KEY_CHECKS = 0;
+TRUNCATE TABLE country;
```

Comment:
this is a lookup table it won't be changed during tests 

it should not be deleted, because after it is deleted all initial data will be removed, but it is required by application

Reply:
Removed this line